lk2nd/device/dts/msm8909/msm8909-zte-sapphire.dts

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <0x1000b 0x6e>, <0x1040b 0x6e>;
+
+	compatible = "qcom,msm8909-qrd", "qcom,msm8909", "qcom,qrd";
+};
+
+&lk2nd {
+	zte-sapphire {
+		model = "ZTE Sapphire";
+		compatible = "zte,sapphire";
+		lk2nd,match-panel;
+		lk2nd,dtb-files = "msm8909-zte-sapphire";
+
+		panel {
```

Comment:
any chance that `qcom,zte-lianchuang-ili9806e-hsd-4p0-fwvga-video` could be another possible panel variant?

Reply:
That is likely